src/GslCore/LegacyParseTypes.fs

```diff
@@ -189,7 +189,7 @@ let private createLegacyPart part =
     | Gene(gw) ->
         convertMods part.x.mods
         >>= (fun mods ->
-            let where = match gw.pos with | Some(p) -> p | None -> emptySourcePosition
+            let where = match gw.positions with | hd::_tl -> hd | [] -> emptySourcePosition
             let genePart = {gene = gw.x.gene; mods = mods; where = where}

```

Comment:
Not a blocker, but at some point we may want to refactor these downstream types to maintain lists of positions as well.

Reply:
Fixed GenePart.  Centralized the formatting too for reporting multiple locations, which removed the need for the system import below too